src/tasks/publish/prettyOutput.ts

```diff
@@ -0,0 +1,152 @@
+import path from 'path'
```

Comment:
Some high level thoughts on this module, potentially a useful exercise for @0xGabi:

What do you think about turning it into a class, that just takes the `txData` on constructor and exposes a number of methods to do pretty printing, etherscan output, receipt output, etc?

It might _feel_ better to use as a single "pretty printer" class than a looser set of utilities.

Reply:
I like the exercise, I try to do the refactor sometime next week?